<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate A Gear!!</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="teethNumberLabel">
     <property name="text">
      <string>Teeth Number</string>
     </property>
     <property name="buddy">
      <cstring>teethNumberSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="teethNumberSpinBox">
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="moduleDoubleSpinBox">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="pressAngleDoubleSpinBox">
     <property name="inputMethodHints">
      <set>Qt::ImhFormattedNumbersOnly</set>
     </property>
     <property name="suffix">
      <string> degree</string>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="filletRadiusDoubleSpinBox">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="shiftLabel">
     <property name="text">
      <string>Shift</string>
     </property>
     <property name="buddy">
      <cstring>shiftDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="shiftDoubleSpinBox">
     <property name="suffix">
      <string/>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QDoubleSpinBox" name="backlashDoubleSpinBox">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Generate</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="moduleLabel">
     <property name="text">
      <string>Module</string>
     </property>
     <property name="buddy">
      <cstring>moduleDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="pressAngleLabel">
     <property name="text">
      <string>Press Angle</string>
     </property>
     <property name="buddy">
      <cstring>pressAngleDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="backlashLabel">
     <property name="text">
      <string>Backlash</string>
     </property>
     <property name="buddy">
      <cstring>backlashDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="filletRadiusLabel">
     <property name="text">
      <string>Fillet Radius</string>
     </property>
     <property name="buddy">
      <cstring>filletRadiusDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="advancedCheckBox">
     <property name="text">
      <string>Advanced</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
